src/app/Scenes/Collect/Collect.tsx

```diff
@@ -0,0 +1,207 @@
+              childrenPosition="left"
+            >
+              <Flex />
+            </ArtworksFilterHeader>
+          </Flex>
+        }
+        onScroll={scrollHandler}
+        refreshControl={RefreshControl}
+        style={{
+          // Extra padding at the bottom of the screen so it's clear that there's no more content
+          paddingBottom: 120,
+        }}
+      />
+
+      <ArtworkFilterNavigator
+        visible={isArtworksFilterModalVisible}
+        mode={FilterModalMode.Collect}
+        exitModal={() => setIsArtworksFilterModalVisible(false)}
+        closeModal={() => setIsArtworksFilterModalVisible(false)}
+      />
+    </>
+  )
+}
+
+export const viewerFragment = graphql`
+  fragment CollectArtworks_viewer on Viewer
+  @refetchable(queryName: "CollectArtwork_RefetchQuery")
+  @argumentDefinitions(
+    count: { type: "Int", defaultValue: 20 }
+    cursor: { type: "String" }
+    input: { type: "FilterArtworksInput" }
+  ) {
+    artworksConnection(
+      first: $count
+      after: $cursor
+      input: $input
+      aggregations: [
+        ARTIST_SERIES
+        COLOR
+        DIMENSION_RANGE
+        LOCATION_CITY
+        MAJOR_PERIOD
+        MATERIALS_TERMS
+        MEDIUM
+        PARTNER
+        PRICE_RANGE
+        ARTIST_NATIONALITY
+      ]
+    ) @connection(key: "CollectArtworks_artworksConnection") {
+      aggregations {
+        slice
+        counts {
+          count
+          name
+          value
+        }
+      }
+      edges {
+        node {
+          id
+          slug
+          image(includeAll: false) {
+            aspectRatio
+          }
+          ...ArtworkGridItem_artwork @arguments(includeAllImages: false)
+        }
+      }
+    }
+  }
+`
+
+export const collectQuery = graphql`
+  query CollectQuery {
+    viewer @required(action: THROW) {
+      ...CollectArtworks_viewer
+    }
+  }
+`
+const CollectQueryRenderer: React.FC = withSuspense({
+  Component: () => {
+    const data = useLazyLoadQuery<CollectQuery>(collectQuery, {})
+
+    return (
+      <ArtworkFiltersStoreProvider>
+        <CollectContent viewer={data.viewer} />
+      </ArtworkFiltersStoreProvider>
+    )
+  },
+  LoadingFallback: () => (
+    <>
+      <Flex px={2}>
+        <Text variant="lg-display">Collect</Text>
+        <Spacer y={0.5} />
+        <Text variant="sm-display">Collect art and design online</Text>
+      </Flex>
+      <ArtworksFilterHeader
+        selectedFiltersCount={0}
+        onFilterPress={() => {}}
+        childrenPosition="left"
+      >
+        <Flex />
+      </ArtworksFilterHeader>
+      <Spacer y={2} />
+      <PlaceholderGrid />
+    </>
+  ),
+  ErrorFallback: (fallbackProps) => {
+    return (
+      <LoadFailureView
+        onRetry={fallbackProps.resetErrorBoundary}
+        showBackButton
+        useSafeArea={false}
+        error={fallbackProps.error}
+        trackErrorBoundary={false}
+      />
+    )
+  },
+})
+
+export const Collect: React.FC = () => {
```

Comment:
Just a suggestion, but I like this pattern because it makes it immediately clear that this is not just a component but a screen.

```suggestion
export const CollectScreen: React.FC = () => {
```